lib/libc/minimal/Kconfig

```diff
@@ -14,31 +14,43 @@ config MINIMAL_LIBC_NON_REENTRANT_FUNCTIONS
 	  option may require an additional memory protection region.
 
 config MINIMAL_LIBC_MALLOC
-	bool "Minimal libc malloc implementation"
+	bool "[DEPRECATED] Minimal libc malloc implementation"
 	default y
+	select COMMON_LIBC_MALLOC
 	help
+	  [DEPRECATED] Use COMMON_LIBC_MALLOC
```

Comment:
After adding a prompt to `COMMON_LIBC_MALLOC`, this needs to `imply COMMON_LIBC_MALLOC` because, otherwise, you would not be able to do `CONFIG_COMMON_LIBC_MALLOC=n` otherwise.

Reply:
oh, very fancy. This is the first I've ever heard of the 'imply' keyword. Fixed (and even tested locally).